### ISSUE TITLE
Additional text file when saving pdf, csv or xls file to file system (iCloud Drive also)

### ISSUE THREAD

**Issue by dahel** [EddyVerbruggen/SocialSharing-PhoneGap-Plugin#919]
Any time I share generated pdf (same with csv, xls) to a iCloud Drive there is additional text file generated with content taken from `message` field passed to `window.plugins.socialsharing.shareWithOptions` method.
For example calling:
```
window.plugins.socialsharing.shareWithOptions(
	 {
		subject: 'subject',
		message: 'some shared file',
		files:  ['file:///[...]/sharedFile.pdf']
	},
		() => {},
		() => {}
);
```
results with exported two files:
one pdf `sharedFile.pdf`
one text file `Text File` with the only content: `some shared file` (the same as value of `message` property passed to method above).

Any ideas how to prevent from `Text File` to be generated?

**Comment by stovmascript** [comment]
In my case (Ionic app), I don't need any `message`, so all I had to do was remove the that key.

```ts
this.socialSharing
	.shareWithOptions({ files })
	.then(() => {})
	.catch(() => {});
```

Same goes for the `.share()` method. I initially had empty strings for the `message` and `subject` arguments, which generated an extra empty `Text File`. Changing those params to `null` fixed the issue:

```ts
this.socialSharing
	.share(null, null, files)
	.then(() => {})
	.catch(() => {});
```

**Comment by Anuj-logiciel** [comment]
Following this issue...

**Comment by Anuj-logiciel** [comment]
Following this issue...

**Comment by stovmascript** [comment]
@Anuj-logiciel You can use the subscribe button on GitHub to follow issues ;) No need to comment and notify everybody.

**Comment by Anuj-logiciel** [comment]
Yep, I know @stovmascript. I just added a comment to prompt contributors to fix this. Thanks for the advice.
